.pipelines/e2e.yaml

```diff
@@ -13,12 +13,7 @@ pr:
       - .pipelines/e2e.yaml
       - .pipelines/templates/e2e-template.yaml
       - e2e
-      - parts/linux
```

Comment:
Sorry. I am not familiar with the syntax here. Could you briefly explain why removing these lines can achieve the purpose `not to trigger e2e test on non e2e file changes`

Reply:
This part defines what files in PR should trigger the pipeline. Currently it can be triggered quite often.

For example, when you change a CSE script. But a new VHD image isn't built in this pipeline. So, the e2e tests are run on master branch VHDs. Which is pointless, as they don't test any changes in the PR submitted.

With the change we will trigger the pipeline only on e2e test file updates.

Reply:
I see. Thanks.